scripts/cstyle.pl

```diff
@@ -590,7 +590,7 @@ ($$)
 	    (/[^!<>=]=[^=\s]/ && !/[^!<>=]=$/)) {
 		# XXX - should only check this for C++ code
 		# XXX - there are probably other forms that should be allowed
-		if (!/\soperator=/) {
+		if (!/\soperator=/ && !/,\s=/) {
```

Comment:
Change the macro should allow you to drop this.

Reply:
no, this just gets further extended later